AY2021S2-CS2103-T14-1/tp
# Add delete alias command

## What this does
This PR
1. adds delete alias command

Closes #88 

## How to test
1. Launch SunRez.
2. Create an alias.
3. Execute the `unalias a/ALIAS_NAME` command with the name of the previously created alias. You should see the alias deleted success message.
4. Execute the alias. You should see a command not found error.
5. Execute the `unalias a/ALIAS_NAME` command with the name of an alias that does not exist. You should see a alias not found error.
